starter/source/materials/mat/mat050/law50_table.F90

```diff
@@ -58,19 +58,19 @@ subroutine law50_table(table  ,nfunc  ,len     ,lmax   ,rate   ,yfac    ,xi
       integer ,intent(inout) :: lmax                      !< max size of function abscissa
       integer ,dimension(nfunc),intent(inout)    :: len   !< size table of all functions
       my_real ,dimension(nfunc) ,intent(in)      :: rate  !< second dimension values
-      my_real ,dimension(nfunc) ,intent(in)      :: yfac  !< scale factors of all functions
       my_real ,dimension(lmax,nfunc) ,intent(in) :: xi    !< x vectors by dimension
       my_real ,dimension(lmax,nfunc) ,intent(in) :: yi    !< y vectors by dimension
       type(table_4d_) ,intent(inout) ::  table            !< table structure
 !-----------------------------------------------
 !     L o c a l   V a r i a b l e s
 !-----------------------------------------------
-      integer :: i,nptx
+      integer :: i,j,nptx,idebug
       my_real ,dimension(:)   ,allocatable :: xf
       my_real ,dimension(:,:) ,allocatable :: yf
 !=======================================================================
 !     create X,Y vectors for all curves and unify all abscissas
 !--------------------------------------------------------
+      idebug = 1
```

Comment:
Hello, 
This should probably be disabled by default

Reply:
yes, thanks :)